src/aleph_client/commands/program.py

```diff
@@ -199,11 +199,11 @@ async def update(
     async with AuthenticatedAlephHttpClient(
         account=account, api_server=sdk_settings.API_HOST
     ) as client:
-        program_message: ProgramMessage = client.get_message(
```

Comment:
@hoh @MHHukiewitz Possible core changes in need.

Reply:
This change looks okay. I don't understand how it could have worked before... :thinking: